docs/docusaurus/docs/reference/api_reference.md

```diff
@@ -1,14 +1,7 @@
 ---
-title: API Documentation
+title: API reference
 ---
 
-:::info WIP
-We are currently working on updating our API Documentation.  These documents are generated from the docstrings of classes and methods that fall under our Public API.
-
-We're excited for contributions to Great Expectations. If you see places where the code or documentation could be improved, please get involved!
-
-You can learn more and get started in the [contributor docs](https://docs.greatexpectations.io/docs/contributing/contributing) or connect on [Slack](https://greatexpectations.io/slack)!
-
-:::
-
+Great Expectations (GX) API reference content is generated from classes and methods docstrings.
```

Comment:
```suggestion
Great Expectations (GX) API reference content is generated from the docstrings of classes and methods.
```
I think this reads a little better (the previous could be mistaken as classes (no docstring) and also methods' docstrings.

I suppose an alternative would be:
```
Great Expectations (GX) API reference content is generated from classes' and methods' docstrings.
```